docs/android/internal-native-dependencies.md

```diff
@@ -22,4 +22,4 @@ return mutableListOf(
 ``` 
 
 In our modules we can pass also variables or even callbacks that we can use for dependency injection.
-Read more in [Using a vertical and dependency injection](https://github.com/kiwicom/mobile/blob/docs/rnkiwimobile/docs/android/verticals-and-dependency-injection.md#using-a-vertical-and-dependency-injection).
\ No newline at end of file
+Read more in [Using a vertical and dependency injection](https://github.com/kiwicom/mobile/blob/docs/rnkiwimobile/docs/android/verticals-and-dependency-injection.md#using-a-vertical-and-dependency-injection).
```

Comment:
@RobinCsl I think the link should be fixed even here (`https://github.com/kiwicom/mobile/blob/master/docs/android/verticals-and-dependency-injection.md#using-a-vertical-and-dependency-injection`).

Reply:
You're right, should be all fixed now. I'll just squash the commits.